### PR TITLE
Mask Detection

### PR DESCRIPTION
**Pull request by siddeshas07**
Face Mask Detection web applicaion built with Flask, Keras-TensorFlow, OpenCV.  It can be used to detect face masks both in images and in real-time video.

### CODE REVIEW

**Comment by SARTHAK2511** [comment]
Thank you for your contribution ✨ . Please give a star to the repo
